From de6a11cc3eeed3f59290351205c75123507d6fe7 Mon Sep 17 00:00:00 2001
From: Lu Hui <luhui@sipeed.com>
Date: Tue, 20 Feb 2024 11:18:29 +0800
Subject: [PATCH 18/18] 
 boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig:
 default use 7inch zct2133v1 panel

---
 .../sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig b/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig
index f685cbe..e13d98a 100644
--- a/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig
+++ b/boards/sg200x/sg2002_licheervnano_sd/sg2002_licheervnano_sd_defconfig
@@ -12,7 +12,7 @@ CONFIG_BOOT_IMAGE_SINGLE_DTB=y
 # CONFIG_FLASH_SIZE_SHRINK is not set
 CONFIG_CP_EXT_WIRELESS=y
 CONFIG_STORAGE_TYPE_sd=y
-CONFIG_MIPI_PANEL_ST7701_HD228001C31=y
+CONFIG_MIPI_PANEL_ZCT2133V1=y
 CONFIG_SENSOR_GCORE_GC4653=y
 CONFIG_UBOOT_2021_10=y
 CONFIG_KERNEL_SRC_5.10=y
-- 
2.40.1

